.rubocop.yml

```diff
@@ -0,0 +1,39 @@
+# use the shared YaST defaults
+inherit_from:
+  /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
+
+# Redundant returns add legibility for developers used to other languages
+Style/RedundantReturn:
+  Enabled: false
+
+# Don't enforce any particular name for block params
+SingleLineBlockParams:
+  Enabled: false
+
+# The general YaST agreement is 100 characters, so it fits into standard
+# resolutions and Github's pull request view. But let's open the threshold a
+# little bit
+Metrics/LineLength:
+  Max: 105
+
+# Enforce if/unless at the end only for really short lines
+Style/IfUnlessModifier:
+  MaxLineLength: 60
+
+# some storage API have size method, but without empty? method
```

Comment:
this is not storage :)

Reply:
Hehe, nothing like use the current config of some new module and not modify comments 👎